### DISCUSSION TITLE
Video Tutorial Please

### DISCUSSION THREAD

**Discussion by BEMC2001** [UltimaHoarder/UltimaScraper#1813] [Q&A]
I get this error, I had 7.1, and everything was fine, there I read that Onlyfans became more strict and that is why the script must be updated, download the most recent and also the most recent version of python, but I get that error, I don't know if the command is wrong when I put it in the CMD, but I just copied and pasted it and it didn't work, please help me with a video tutorial
![Screenshot_20211220-183520_Telegram](https://user-images.githubusercontent.com/96454289/146856118-b26c60bc-589a-49c7-81d6-0cc068ed1326.jpg)

**Comment by jozipaulo** [comment]
This is an old version of the scraper program. The latest is 7.6.1 at this time (26 Dec 2021)
You can locate the latest version here - https://github.com/DIGITALCRIMINALS/OnlyFans
Looks like this DIGITALCRIMINAL not DIGITALCRIMINALS repository is no longer active.
